meinberlin/apps/notifications/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+
+
+class NotificationSettings(models.Model):
+    email_fields = [
+        "email_newsletter",
+        "notify_followers_phase_started",
+        "notify_followers_phase_over_soon",
+        "notify_followers_event_upcoming",
+        "notify_creator",
+        "notify_creator_on_moderator_feedback",
+        "notify_initiators_project_created",
+        "notify_moderator",
+    ]
+    user = models.OneToOneField(
+        "meinberlin_users.User",
+        on_delete=models.CASCADE,
+        related_name="notification_settings",
+    )
+    email_newsletter = models.BooleanField(default=False)
+
+    """
+    Notification fields are being used to check if a notification should be sent
+    via email.
+    """
+    notify_followers_phase_started = models.BooleanField(default=True)
+    notify_followers_phase_over_soon = models.BooleanField(default=True)
+    notify_followers_event_upcoming = models.BooleanField(default=True)
+    notify_creator = models.BooleanField(default=True)
+    notify_creator_on_moderator_feedback = models.BooleanField(default=True)
+    notify_initiators_project_created = models.BooleanField(default=True)
+    notify_moderator = models.BooleanField(default=True)
+
+    """
+    Tracked fields are being used to check if a notification should show in
+    the activity feed.
+    """
+    track_followers_phase_started = models.BooleanField(default=True)
+    track_followers_phase_over_soon = models.BooleanField(default=True)
+    track_followers_event_upcoming = models.BooleanField(default=True)
```

Comment:
by default they are all enabled?

Reply:
Yes, these are no email notifications, they will just determine whether these will be shown in the activity feed or not so I thought it would be fine to start with them enabled. There's also a comment like that in taiga.